.circleci/config.yml

```diff
@@ -1,12 +1,15 @@
 version: 2.1
 orbs:
   node: circleci/node@1.1.6
+  dotenv: anilanar/dotenv@volatile
 jobs:
   build-and-test:
     executor:
       name: node/default
     steps:
       - checkout
+      - dotenv/source:
```

Comment:
For a cleaner github history it'd be good to remove the commits about adding and removing the dotenv orb mention - a github rebase -interactive should help do that.